taxi/models.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+
+from django.db import models
+from django.db.models import CASCADE
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(
+        max_length=63,
+        unique=True,
+    )
+
+    class Meta:
+        ordering = ["license_number", ]
+
+    def __str__(self):
+        return self.license_number
```

Comment:
There's no need to overuse f-string. Since `self.license_number` is a string, you can return it directly without using f-string.